nbconvert/exporters/templateexporter.py

```diff
@@ -163,6 +163,31 @@ def _template_file_default(self):
     
     #Extension that the template files use.
     template_extension = Unicode(".tpl").tag(config=True, affects_environment=True)
+    
+    exclude_input = Bool(False,
+        help = "This allows you to exclude code cell inputs from all templates if set to True."
```

Comment:
When @fperez and I spoke Monday, the suggestion was that this was a use case that people might want (extracting only the markdown of their document). 

Furthermore, I don't think it's intrinsically harmful to include.

One potential use case would be to write a report with the figures saved to disk with explicit invocations of the figures inline with the text (or, for example, in raw cells with figure calls). That would allow people to specify labels and captions from within their notebooks which would solve, for example, [this post to the jupyter group](https://groups.google.com/forum/#!topic/jupyter/l_GhfzAMJDc).